draft-ferrieuxhamchaoui-quic-lossbits.md

```diff
@@ -431,6 +432,24 @@ Specification: Indicates that the endpoint supports loss bits. An endpoint that
    advertises this transport parameter with value 1 can also send loss bits.
 
 
+# Change Log
+
+## Since version 02
+
+* Add QUIC v1 negotiation: transport parameter, header protection change, IANA
```

Comment:
More than just negotiation was added.  This list is missing short packet header change (R bits to QL bits).

Reply:
Sure.  I'll add that.